### PR TITLE
Add short url for all api resources

### PR DESCRIPTION
**Pull request by ghost**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json [hunk 1]
```diff
@@ -1443,9 +1443,9 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/apis/sql/databases": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/sqlDatabases": {
```
Comment (yungezz) [inline specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json]: this is breaking change. pls add a new api version.

**Review by yungezz — CHANGES_REQUESTED** [review]

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json (hunk 1 above)
Comment (bizarreWizard) [reply specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json]: Hi, How can I add a new api version?

Comment (yungezz) [reply specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json]: add a new swagger file with new tag under azure-rest-api-specs/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable or preview, such as 2019-07-01 etc. sample azure-rest-api-specs/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/

Comment (bizarreWizard) [reply specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json]: Hi @yungezz my alias is chuyu. Can you please ping me, let's sync on Teams.

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2019-08-01/cosmos-db.json [hunk 2]
```diff
@@ -0,0 +1,6108 @@
...
+              "$ref": "#/definitions/ThroughputSettingsGetResults"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "SqlResources_UpdateSqlContainerThroughput",
+        "x-ms-examples": {
+          "CosmosDBSqlContainerThroughputUpdate": {
+            "$ref": "./ex
```
[the remark this answers is not in the document]
Comment (ryansbenson) [reply specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2019-08-01/cosmos-db.json]: For context from our Teams chat:

I know your previous PR separated the get response and put content types as different type definitions. These should be the same type definition as the RPC has a -get, then update, then put- pattern which should be supported. Not being the same type definitions gets you into an odd place when it comes to testing/validating the swagger. For example:

    ThroughputSettingsUpdateParameters contains ThroughputUpdateProperties which contains ThroughputSettingsResource
the above is for put

for get the type definition is:

    ThroughputSettingsGetResults contains ThroughputSettingsGetProperties contains BOTH ThroughputSettingsResource and ExtendedThroughputSettingsProperties

So the swagger effectively states ExtendedThroughputSettingsProperties could come back from a GET but there is no way to set them

**Comment by bizarreWizard** [comment]
Please hold merge

**Comment by yungezz** [comment]
Hi @bizarreWizard what's the plan to proceed this PR? is it ok to close it for now and reopen you're ready for merge?

**Comment by bizarreWizard** [comment]
> Hi @bizarreWizard what's the plan to proceed this PR? is it ok to close it for now and reopen you're ready for merge?

The PR should be able to merge on the beginning of Sep. We are waiting for prod deployment.

**Comment by bizarreWizard** [comment]
Please merge the PR, thanks!

**Review by yungezz — APPROVED** [review]

**Pull request merged**